package.json

```diff
@@ -21,5 +21,9 @@
   "files": [
     "github-btn.html",
     "LICENSE.md"
-  ]
+  ],
+  "devDependencies": {
+    "gulp": "^3.8.11",
+    "gulp-html-minifier": "^0.1.6",
```

Comment:
This comma breaks stuff since it's not valid :P

Don't forget to squash when you are done.